src/jbod/disks.rs

```diff
@@ -136,13 +136,11 @@ pub mod DiskShelf {
     fn get_disk_serial(disk: String) -> String {
         let res = fs::read(disk + "/vpd_pg80");
         let content = match res {
-            Ok(c) => { c.to_vec() },
-            Err(_err) => { "N/A".as_bytes().to_vec() }
+            Ok(c) => c.to_vec(),
+            Err(_err) => "N/A".as_bytes().to_vec(),
         };
 
-        unsafe {
-            String::from_utf8_unchecked(content.to_vec()).to_string()
-        }
+        unsafe { String::from_utf8_unchecked(content.to_vec()).to_string() }
```

Comment:
Small NIT maybe here we can remove an extra `to_vec` since `content` is already a `Vec<u8>`
```suggestion
        unsafe { String::from_utf8_unchecked(content).to_string() }
```

Reply:
It is not the intention of this patch, as stated above, I just run "fmt", these changes can be addressed in a different PR.